src/mcs/EmpiricalDistribution.jl

```diff
@@ -33,58 +29,36 @@ struct EmpiricalDistribution{T}
-
-If an XLS or XLSX file is given, the `value_col` and `prob_col` should be fully specified
-Excel data ranges, e.g., "Sheet1!A2:A1002", indicating the values to extract from the file.
-"""
-function EmpiricalDistribution(filename::AbstractString,
-                               value_col::Union{Symbol, AbstractString, Int},
-                               prob_col::Union{Void, Symbol, AbstractString, Int}=nothing;
-                               value_type::DataType=Any)
-    probs = nothing
-    ext = splitext(lowercase(filename))[2]
-
-    if ext == ".csv"
-        df = DataFrame(load(filename))
-        values = isa(value_col, Symbol) ? df[value_col] : df.columns[value_col]
-        
-        if prob_col != nothing
-            probs = Vector{Float64}(isa(prob_col, Symbol) ? df[prob_col] : df.columns[prob_col])
-        end
-    elseif ext in (".xls", ".xlsx", ".xlsm")
-
-        if Pkg.installed("ExcelReaders") == nothing
-            error("""You must install ExcelReaders to read Excel files: run Pkg.add("ExcelReaders")""")
-        end
+# """
+# EmpiricalDistribution(obj::Any,
+#                       value_col::Union{Symbol, String, Int},
+#                       prob_col::Union{Void, Symbol, String, Int}=nothing;
+#                       value_type::DataType=Any)
+
+# Load empirical values from an object returned by load() and generate a distribution. 
+# Typical usage might be:
+
+# `d = EmpiricalDistribution(load("foo.xlsx", otherpars=...), :valcolname, :probcolname)`
+
+# The `value_col` identifies by the column name or integer index for the values 
+# to use, and the optional `prob_col` identifies the column name or integer index for the
+# probabilities to use. If `prob_col` is not provide, equal probabilities are assumed 
+# for each value.
+# """
+# function EmpiricalDistribution(obj::Any,
+#                                value_col::Union{Symbol, AbstractString, Int},
+#                                prob_col::Union{Void, Symbol, AbstractString, Int}=nothing;
+#                                value_type::DataType=Any)
+#     probs = nothing
+
+#     values = isa(value_col, Symbol) ? df[value_col] : df.columns[value_col]
         
-        f = openxl(filename)
-        data = Array{value_type, 2}(readxl(f, value_col))
-        values = data[:, 1]
+#     if prob_col != nothing
+#         probs = Vector{Float64}(isa(prob_col, Symbol) ? df[prob_col] : df.columns[prob_col])
+#     end
 
-        if prob_col != nothing
-            data = Array{Float64, 2}(readxl(f, prob_col))
-            probs = data[:, 1]
-        end
-    else
-        error("Unrecognized file extension '$ext'. Must be .csv, .xls, .xlsx, or .xlsm")
-    end
-    return EmpiricalDistribution(values, probs)
-end
-
-# If a column is not identified, use the first column.
-function EmpiricalDistribution(filename::AbstractString)
-    return EmpiricalDistribution(filename, 1)
-end
+#     return EmpiricalDistribution(values, probs)
+# end
```

Comment:
Why is this part of the PR? Things like this should really go in their own PR.

Reply:
It's true that this is separate from the MCS stuff. It just gets tedious stashing / branching / unstashing for every small fix...

Reply:
On the other hand, it's all withing the MCS directory and felt "close enough" to include it. :~)